ooni/errors.py

```diff
@@ -26,6 +27,8 @@
     (CancelledError, 'task_timed_out'),
     (gaierror, 'address_family_not_supported_error'),
     (DNSLookupError, 'dns_lookup_error'),
+    (DNSNameError, 'dns_name_error'),
+    (DNSServerError, 'dns_server_failure'),
```

Comment:
The reason why I chose to go for this mapping instead of the twisted one is that it seemed to be more "universal" since it's the terminology that is used inside of [RFC1035](https://www.ietf.org/rfc/rfc1035.txt) for RCODE 2 ("Server failure - The name server was unable to process this query due to a problem with the name server").